README.md

```diff
@@ -57,6 +57,10 @@ For instructions on installing the root server, see [the page on installing a ne
 
 CHANGELIST
 ----------
+***Version 2.14.5* ** *- UNRELEASED*
+
+- Added Other tab from admin user interface to translatable strings.
```

Comment:
Change to `The "Other" tab in the meeting editor user interface can now be translated.`

Then feel free to merge.